shared/src/main/scala/bloop/io/OsEnv.scala

```diff
@@ -0,0 +1,41 @@
+package bloop.io
+
+object OsEnv {
```

Comment:
Should we maybe add the methods to `SystemProperties` or `Environment` object? Probably the latter one would better.

Reply:
I think it needs to be in the shared project, since there are issues in the frontend and the backend.
I tried to add it to frontend bloop.util.Environment, but it's then not visible in the backend.
I maybe should rename it to Environment, though.

Reply:
I haven't noticed it's in shared :sweat: 

Maybe let's just rename it to `OsEnvironment` ?

Reply:
Agreed (rename to OsEnvironment).   I'm working on this, but only got Intellij working a few minutes ago.